### DISCUSSION TITLE
Installation in containers environment

### DISCUSSION THREAD

**Discussion by b3ko91** [teslamate-org/teslamate#3840] [General]
Hello,

Does it's possible to use postgres:latest and Mosquitto:latest in Teslamate docker environment ?
In installation documentation, the sample docker-compose.yml define postgres:15 and eclipse-mosquitto:2
Thanks for responses.

Olivier

**Comment by StarkITA** [comment]
Hello,

why do you want to use the latest version? Usually it's best practice to stick to versions that are sure to work :)

**Comment by brianmay** [comment]
Also upgrading postgres, at least between major versions, requires manual work. It isn't something you want to have updated unexpectedly.
